src/hotspot/share/nmt/memTracker.hpp

```diff
@@ -269,6 +277,14 @@ class MemTracker : AllStatic {
   // and return true; false if not found.
   static bool print_containing_region(const void* p, outputStream* out);
 
+  // Same as MutexLocker but can be used during VM init while single threaded and before mutexes are ready or current thread has been assigned.
+  // Performs no action during VM init.
+  class NmtVirtualMemoryLocker: StackObj {
+      ConditionalMutexLocker _cml;
+  public:
+      NmtVirtualMemoryLocker(): _cml(NmtVirtualMemory_lock, _done_bootstrap, Mutex::_no_safepoint_check_flag){}
```

Comment:
Indented 4, but HotSpot style is indent 2.

Reply:
Updated. Thank you!